cmd/environment.go

```diff
@@ -41,8 +40,6 @@ func (s *EnvSettings) AddBaseFlags(fs *pflag.FlagSet) {
 // AddFlags binds flags to the given flagset.
 func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	s.AddBaseFlags(fs)
-	fs.StringVar(&s.KubeConfigFile, "kubeconfig", "", "path to the kubeconfig file")
-	fs.StringVar(&s.KubeContext, "kube-context", s.KubeContext, "name of the kubeconfig context to use")
```

Comment:
Is this needed so that it shows the flags for help?

Reply:
I hadn't thought of that.  That is a problem.  I guess your suggestion is best.  I'll these line back but won't make them stored in anything real.

Reply:
So keeping the flags made me think that we may as well support them properly as you had originally done.  The latest version of the PR leaves everything as it was except that it also reads the HELM_KUBECONTEXT variable in case it is set by helm.

So now `--kube-context` and `--kubeconfig` will work both when `2to3` is used as a plugin or directly as a binary.